test/integration/imagequalify_admission_test.go

```diff
@@ -0,0 +1,83 @@
+package integration
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	configapi "github.com/openshift/origin/pkg/cmd/server/api"
+	testutil "github.com/openshift/origin/test/util"
+	testserver "github.com/openshift/origin/test/util/server"
+	kapi "k8s.io/kubernetes/pkg/apis/core"
+)
+
+func TestImageQualifyAdmission(t *testing.T) {
+	pluginFile, err := ioutil.TempFile("", "admission.yaml")
+	if err != nil {
+		t.Fatal(err)
+	}
+	defer os.Remove(pluginFile.Name())
+
+	err = ioutil.WriteFile(pluginFile.Name(), []byte(`
+apiVersion: v1
+kind: ImageQualifyConfig
+rules:
+- pattern: busybox
+  domain: domain1.com
+- pattern: mylib/*
+  domain: mydomain.com
+`), os.FileMode(0644))
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	masterConfig, err := testserver.DefaultMasterOptions()
+	if err != nil {
+		t.Fatalf("error creating config: %v", err)
+	}
+	defer testserver.CleanupMasterEtcd(t, masterConfig)
+	masterConfig.AdmissionConfig.PluginConfig = map[string]configapi.AdmissionPluginConfig{
+		"openshift.io/ImageQualify": {
```

Comment:
Refer to `openshift/origin/pkg/image/admission/imagequalify/api.PluginName`.

Reply:
No, if the name changes we want something in tree to smack us since we broke compatibility